src/humanize/time.py

```diff
@@ -210,3 +233,253 @@ def naturaldate(value):
     if delta.days >= 5 * 365 / 12:
         return naturalday(value, "%b %d %Y")
     return naturalday(value)
+
+
+def _quotient_and_remainer(value, divisor, unit, minimum_unit, suppress):
+    """Divide ``value`` by ``divisor`` returning the quotient and
+       the remainer as follows:
+
+       If unit is minimum_unit, makes the quotient a float number
```

Comment:
There's a few more like this to do too:
```suggestion
       If ``unit`` is ``minimum_unit``, makes the quotient a float number
```